src/Microsoft.Azure.WebJobs.Host/Tables/TableExtension.cs

```diff
@@ -306,43 +305,6 @@ public JObjectBuilder(TableExtension bindingProvider)
             }
         }
 
-        // IStorageTable --> IQueryable<T>
-        // ConverterManager's pattern matcher will figure out TElement. 
-        private class TableToIQueryableConverter<TElement> :
```

Comment:
Can you explain again why we can no longer support IQueryable? What if we wanted to retain this capability - how hard would it be?